k8s/services/rabbitmq/helm-3.4.cue

```diff
@@ -0,0 +1,294 @@
+		registry:   string | *"docker.io"
+		repository: string | *"bitnami/rmq-default-credential-updater"
+		tag:        string | *"1.0.2-scratch-r21"
+		digest:     string | *""
+		pullSecrets: [...v1.#LocalObjectReference]
+	}
+
+	clusterOperator: {
+		image: {
+			registry:   string | *"docker.io"
+			repository: string | *"bitnami/rabbitmq-cluster-operator"
+			tag:        string | *"2.2.0-scratch-r7"
+			digest:     string | *""
+			pullPolicy: v1.#enumPullPolicy | *"IfNotPresent"
+			pullSecrets: [...v1.#LocalObjectReference]
+		}
+		replicaCount: uint | *1
+		topologySpreadConstraints: []
+		schedulerName:                 string | *""
+		terminationGracePeriodSeconds: string | *""
+		livenessProbe:                 v1.#Probe | *{
+			enabled:             true
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		readinessProbe: v1.#Probe | *{
+			enabled:             true
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		startupProbe: v1.#Probe | *{
+			enabled:             false
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		customLivenessProbe:  v1.#Probe | *{}
+		customReadinessProbe: v1.#Probe | *{}
+		customStartupProbe:   v1.#Probe | *{}
+		resources:            v1.#ResourceRequirements | *{
+			limits: {}
+			requests: {}
+		}
+		podSecurityContext: v1.#PodSecurityContext | *{
+			enabled: true
+			fsGroup: 1001
+		}
+		containerSecurityContext: v1.#PodSecurityContext | *{
+			enabled:                true
+			runAsUser:              1001
+			runAsNonRoot:           true
+			readOnlyRootFilesystem: true
+		}
+		command: v1.#Command | *[]
+		args:    v1.#Args | *[]
+		hostAliases: []
+		podLabels:             k8s.#Labels
+		podAnnotations:        k8s.#Annotations
+		podAffinityPreset:     string | *""
+		podAntiAffinityPreset: string | *"soft"
+		nodeAffinityPreset:    v1.#Affinity
+		affinity:              v1.#Affinity
+		nodeSelector:          v1.#NodeSelector
+		tolerations: [...v1.#Toleration]
+		updateStrategy: {
+			type: "RollingUpdate"
+		}
+		priorityClassName: string | *""
+		lifecycleHooks: {}
+		containerPorts: {
+			metrics: k8s.#Port | *9782
+		}
+		extraEnvVars: []
+		extraEnvVarsCM:     string | *""
+		extraEnvVarsSecret: string | *""
+		extraVolumes: []
+		extraVolumeMounts: []
+		sidecars: []
+		initContainers: []
+		rbac: {
+			create: bool | *true
+		}
+		serviceAccount: #ServiceAccount
+		metrics:        #Metrics
+	}
+
+	msgTopologyOperator: {
+		image: {
+			registry:   "docker.io"
+			repository: "bitnami/rmq-messaging-topology-operator"
+			tag:        "1.10.3-scratch-r1"
+			digest:     ""
+			pullPolicy: "IfNotPresent"
+			pullSecrets: [...v1.#LocalObjectReference]
+		}
+		replicaCount: uint | *1
+		topologySpreadConstraints: []
```

Comment:
what's the schema? maybe [...string]

if it's not know we can leave it as anything [..._]